client/src/components/Utilities/GetSequenceIds/GetSequenceIds.tsx

```diff
@@ -1,22 +1,150 @@
-import { TextField } from "@material-ui/core";
-import { useState } from "react";
+import {
+  Button,
+  Link,
+  makeStyles,
+  TextField,
+  Typography,
+} from "@material-ui/core";
+import React, { useEffect, useState } from "react";
 import "./GetSequenceIds.scss";
```

Comment:
This line should be removed (except let me work on styling stuff right now, in case we need to leave this)